sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -278,6 +278,7 @@ object FunctionRegistry {
     // misc functions
     expression[Crc32]("crc32"),
     expression[Md5]("md5"),
+    expression[Murmur3Hash]("murmur3_hash"),
```

Comment:
maybe we can have a flag to control this -- when in hive compatibility test, fall back to Hive's, and otherwise our own?